### ISSUE TITLE
read.section() missing codes NITRAT, NH4, CFC-12, SF6, TCARBN, ALKALI, PH_TOT, PH_TMP

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1227]
Sample data from https://cchdo.ucsd.edu/cruise/18HU20150504; link is https://cchdo.ucsd.edu/data/12503/18HU20150504.exc.csv and I get as follows:
```R
library(oce)
d <- read.oce("https://cchdo.ucsd.edu/data/12503/18HU20150504.exc.csv")
summary(d[["station", 1]])
```
yielding as below
```
CTD Summary
-----------

* Start time:          2015-05-05 00:10:00
* Station:             6
* Location:            44.268N  63.316W 
* Water depth:         -999
* Data

                             Min.    Mean   Max.   Dim. OriginalName
    pressure [db]            2.5     56.075 146.7  24   CTDPRS      
    temperature [°C, ITS-90] -0.7456 2.8206 9.8766 24   CTDTMP      
    salinity [PSS-78]        30.777  32.034 34.869 24   CTDSAL      
    salinityBottle [PSS-78]  30.805  32.354 34.866 24   SALNTY      
    oxygen [μmol/kg]         148.8   277.3  390.7  24   CTDOXY      
    silicate [μmol/kg]       0.05    4.653  14.59  24   SILCAT      
    NITRAT [μmol/kg]         0.49    6.3    17.59  24   NITRAT      
    nitrite [μmol/kg]        0.01    0.056  0.1    24   NITRIT      
    phosphate [μmol/kg]      0.25    0.696  1.31   24   PHSPHT      
    NH4 [μmol/kg]            0.48    0.71   1.1    24   NH4         
    CFC-12 [PMOL/KG]         NA      NA     NA     24   CFC-12      
    SF6 [FMOL/KG]            NA      NA     NA     24   SF6         
    TCARBN [μmol/kg]         NA      NA     NA     24   TCARBN      
    ALKALI [μmol/kg]         NA      NA     NA     24   ALKALI      
    PH_TOT                   NA      NA     NA     24   PH_TOT      
    PH_TMP [DEG C]           9       9      9      24   PH_TMP      
...
```

**Comment by dankelley** [comment]
Also (and this is why I noticed)
```R
> plot(d)
Error in pretty.default(c(0, p.max), n = min(200, floor(p.max/dp))) : 
  invalid 'n' argument
```

**Comment by dankelley** [comment]
Fixed in `develop` commit 040bd13ece7c3aa177b2105a9484ba06b9f6bbaa, viz
```R
                              Min.    Mean   Max.   Dim. OriginalName
    pressure [db]             2.5     56.075 146.7  24   CTDPRS      
    temperature [°C, ITS-90]  -0.7456 2.8206 9.8766 24   CTDTMP      
    salinity [PSS-78]         30.777  32.034 34.869 24   CTDSAL      
    salinityBottle [PSS-78]   30.805  32.354 34.866 24   SALNTY      
    oxygen [μmol/kg]          148.8   277.3  390.7  24   CTDOXY      
    silicate [μmol/kg]        0.05    4.653  14.59  24   SILCAT      
    nitrate [μmol/kg]         0.49    6.3    17.59  24   NITRAT      
    nitrite [μmol/kg]         0.01    0.056  0.1    24   NITRIT      
    phosphate [μmol/kg]       0.25    0.696  1.31   24   PHSPHT      
    ammonium [μmol/kg]        0.48    0.71   1.1    24   NH4         
    CFC12 [PMOL/KG]           NA      NA     NA     24   CFC-12      
    SF6 [FMOL/KG]             NA      NA     NA     24   SF6         
    totalCarbon [μmol/kg]     NA      NA     NA     24   TCARBN      
    totalAlkalinity [μmol/kg] NA      NA     NA     24   ALKALI      
    pHTotal                   NA      NA     NA     24   PH_TOT      
    pHTemperature [DEG C]     9       9      9      24   PH_TMP      
```

**Comment by dankelley** [comment]
Oh, hang on, I need fix the units of some of these now, too.  ... now, `develop` commit 29886e4db2fb62210fa2823dabed3597618b6331 yields
```R
                              Min.    Mean   Max.   Dim. OriginalName
    pressure [db]             2.5     56.075 146.7  24   CTDPRS      
    temperature [°C, ITS-90]  -0.7456 2.8206 9.8766 24   CTDTMP      
    salinity [PSS-78]         30.777  32.034 34.869 24   CTDSAL      
    salinityBottle [PSS-78]   30.805  32.354 34.866 24   SALNTY      
    oxygen [μmol/kg]          148.8   277.3  390.7  24   CTDOXY      
    silicate [μmol/kg]        0.05    4.653  14.59  24   SILCAT      
    nitrate [μmol/kg]         0.49    6.3    17.59  24   NITRAT      
    nitrite [μmol/kg]         0.01    0.056  0.1    24   NITRIT      
    phosphate [μmol/kg]       0.25    0.696  1.31   24   PHSPHT      
    ammonium [μmol/kg]        0.48    0.71   1.1    24   NH4         
    CFC12 [pmol/kg]           NA      NA     NA     24   CFC-12      
    SF6 [fmol/kg]             NA      NA     NA     24   SF6         
    totalCarbon [μmol/kg]     NA      NA     NA     24   TCARBN      
    totalAlkalinity [μmol/kg] NA      NA     NA     24   ALKALI      
    pHTotal                   NA      NA     NA     24   PH_TOT      
    pHTemperature [°C]        9       9      9      24   PH_TMP  
```
